### DISCUSSION TITLE
buddhism/zen_reader/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#44] [General]
# buddhism/zen_reader/

https://shanenull.com/buddhism/zen_reader/

**Comment by shane0** [comment]
THE NATURE OF THINGS

THE NATURE of things is fundamental perpetual silence, open and clear, without limits or boundaries.
If you place your mind in the midst of grasping and rejecting, you will be under the influence of those two states.
If you concentrate, enter trance, and sit in meditation, focusing on an objective, setting your mind on awareness and contemplation, practicing the Way like a mechanical mannequin, when will you ever arrive at the goal?
All things are fundamentally empty; there is nothing to stick to. Objects are like floating clouds, certain to disperse. When you realize the basic emptiness of fundamental essence, that will be like a fever's breaking. Don't speak of it to the ignorant, or they'll beat your body to pieces.

Pao-chih
